test/integration/my-stuff.test.js

```diff
@@ -55,13 +55,15 @@ describe('www-integration my_stuff', () => {
         await driver.sleep(6000);
         let gui = await findByXpath('//div[@class="guiPlayer"]');
         await expect(gui.isDisplayed());
+        await driver.sleep(1500);
```

Comment:
Why do these need to wait if the test is already over?

Reply:
We shouldn't, but for some reason when running on saucelabs it seems to start the next test before the expect() line returns.  The tests pass with the driver.sleep and fail without them.

Reply:
Is `gui.isDisplayed` an async function? I assume it is since it has to ask saucelabs for the answer. If that's what we're trying to wait on with the `await`, then I don't think `await expect(gui.isDisplayed())` will work, since it's going to await the `expect`, and `expect` isn't async. So basically if `gui.isDisplayed` is what we're waiting for, we should separately `await` that, and then run the `expect` on the result.  Otherwise `expect` will immediately return, and the next test would start.

Reply:
And I should say, it would be better to await gui.isDisplayed than to await the flat 1.5s because it's possible gui.isDisplayed will finish sooner than 1.5s, and then our tests can finish faster (e.g. when we run tests locally).  And in the case that it takes longer than 1.5s to return, the test will still pass.